packages/block-library/src/post-content/edit.js

```diff
@@ -32,9 +32,9 @@ function ReadOnlyContent( { userCanEdit, postType, postId } ) {
 			<Warning>{ __( 'This content is password protected.' ) }</Warning>
 		</div>
 	) : (
-		<div { ...blockProps }>
-			<RawHTML key="html">{ content?.rendered }</RawHTML>
-		</div>
+		<RawHTML key="html" { ...blockProps }>
```

Comment:
This causes and error in the console because it's trying to set a ref on an functional component which isn't possible.

Reply:
Can you use `dangerouslySetInnerHTML` on the `div` instead?

Reply:
Oh yeah, that seems to work. What do you think?